test/test_manifest.json

```diff
@@ -5954,6 +5954,15 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {
+	   "id": "issue6127_scale4",
+	   "file": "pdfs/issue6127.pdf",
+	   "md5": "65ef9572bddef5fddfaf7172f10a0df2",
+	   "rounds": 1,
+	   "lastPage": 1,
+	   "scale": 4,
+	   "type": "eq"
+    },
```

Comment:
https://github.com/mozilla/pdf.js/pull/14534#discussion_r800173739 unfortunately still applies here, since you've mixed tabs and spaces.

Please note that we *only* use spaces for indentation everywhere in the code-base. It might help to setup the EditorConfig integration for your code-editor to help with formatting, see e.g. https://editorconfig.org/#pre-installed or https://editorconfig.org/#download

---

> Setting to a value >1 results in a larger image for the browsertest.

*Just a FYI:* It'll also be possible to set the `scale` to a value *below* `1`, since one here simply means 100% (e.g. in the viewer).